src/CMakeLists.txt

```diff
@@ -54,9 +54,17 @@ endif()
 set_target_properties(mercury_shared
     PROPERTIES OUTPUT_NAME mercury
 )
-set_target_properties(mercury_static
-    PROPERTIES OUTPUT_NAME mercury
-)
+# For now, under Windows we rename the static library since the import
+# library for the DLL will have the same name and overwrite it.
+if (MSVC)
+    set_target_properties(mercury_static
+        PROPERTIES OUTPUT_NAME mercury_static
```

Comment:
Incidentally, by default the name if you don't explicitly set it here will be `mercury_static` because that is what the target is called. I'm fine with being explicit here but just FYI.